gvmtools/pyshell.py

```diff
@@ -163,8 +140,15 @@ def main():
         enter_interactive_mode(global_vars)
 
     if script_and_interactive or only_script:
+        if only_script:
+            print(
+                'Using gvm-pyshell for running scripts only is deprecated. '
```

Comment:
```suggestion
                'Using gvm-pyshell for running scripts is deprecated. '
```

Reply:
It's and will still be possible to run scripts within pyshell. You can run a script and afterwards go into the shell. But running only a script should not be done with pyshell anymore.

Reply:
Ah, I understand.